### PR TITLE
Pull request for Blog app - controllers

### PR DESCRIPTION
**Pull request by zuhebahmed88091**
Dear reviewer,

## In this PR:

---

- [ ] Made sure that the app handles the URLs used in the provided wireframes.

- [ ] For each URL created:

    - A route.
    - An action in the correct controller.
    - A view file.
    - View files included only placeholders written in plain HTML.

Made sure that added code to handle only the 4 required URLs.

req

Thank you!

Regards,

@zuhebahmed88091

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show], controller: 'user_posts'
```
Comment (codepantha) [inline config/routes.rb]: - [ ] By specifying `controller: 'user_posts'`, you are pointing Rails to `UserPostsController` which doesn't exist thereby causing the error shown below 👇. Please take it off and Rails will handle the routing correctly 👍 

image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Hi @zuhebahmed88091 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Set up controllers with the `#index` and `#show` actions 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (zuhebahmed88091) [reply config/routes.rb]: Okay.. done that

**Review by DammyShittu — APPROVED** [review]
Hi @zuhebahmed88091,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zuhebahmed88091** [comment]
Thank you @DammyShittu @codepantha

**Pull request merged**